ArrayUtils.js

```diff
@@ -1,21 +1,84 @@
 class ArrayUtils{
 
-    isEmpty(array){return false;}
 
-    append(original, value){return original;}
+     isEmpty(array){
+        if (array.length !== 0){
+        return false;
+        }
+    }
+    
+    append(original, value){
+        for (let i = 0; i <= original.length; i ++){
+            if(i === original.length -1){
+                original += value
+            }
+        }
+
+        return original
+    }
 
-    clone(original){return original;}
+    clone(original){
+        const array = []
+        for(let i = 0; i <= original.length; i++){
+            array += orginal
+        }
+        return array
+    }
 
-    subArray(original, from, to){return original;}
+    subArray(original, from, to){
+        let output = ""
+        for(let i = 0; i <= original.length; i ++){
+            for(let j = from[i]; j <= to[i].length; j++){
+                output += j
+            }
+        }
+        
+        return output
+    }
 
-    equals(arr1, arr2){return false;}
+    equals(arr1, arr2){
+        if(arr1 !== arr2){
+        return false;
+        }
+    }
 
-    fill(original, value){}
```

Comment:
Should not be adding elements to an array, should be updating the existing values (-11)